docker-compose.yml

```diff
@@ -12,6 +13,24 @@ services:
     environment:
       - SERVER_PORT=8080
       - SPRING_PROFILES_ACTIVE=dev
+      - HMPPS_AUTH_URL=https://sign-in-dev.hmpps.service.justice.gov.uk/auth
```

Comment:
Minor - I wonder if there's a docker alternative, so we're not relying on that service.

Reply:
Yes, there totally is. We can bring hmpps-auth in as a docker container and run it locally 👍 
Its just that I wanted to get something up quickly in this case - we can defo add to this over time 👍